source/target/freescale/rapid_iot/target_reset.c

```diff
@@ -38,6 +38,8 @@
 #define KW41         1
 
 static uint32_t mdm_id;
+extern char *board_id;
+char *board_id_kw41z = "0234";
```

Comment:
Should we put all board IDs in the board source file instead; please see mkit_dk_dongle_nf5x.c.

Reply:
This file is shared by the hexiwear board as well. Moving this to rapidiot board file will break the build for hexiwear.

Reply:
The problem is that the logic to detect the connected target is in this `targe_reset.c` file. If we moved the detect logic to the board source file, it would have to be duplicated for RapidIoT and Hexiwear. So no perfect solution here.

Reply:
@brianesquilona. As Chris mentioned this would cause code duplication. Can we stick with the current implementation?

Reply:
@mmahadevan108 , This PR will be merged in a bit